modules/apps/commerce/commerce-cart-content-web/src/main/resources/META-INF/resources/js/cart_total/transition.js

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
Instead of creating a copy if `/cart_mini/transition.js`, move the js file under `/js` folder and export it like this `export {default as cartMiniTransition} from './cart_mini/transition';` in the` /js/index.js` file